packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/sections/components/InvoicePreviewMonite.module.css

```
@@ -0,0 +1,369 @@
+  --border-radius: var(--spacing-xs);
+}
+
+.invoicePreviewMonite {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  min-height: 29.7cm;
+  padding: 2.5rem;
+  font-family: 'Inter', 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: Helvetica

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)